.github/workflows/build_and_publish.yaml

```
@@ -57,7 +57,7 @@ jobs:
         python setup.py sdist bdist_wheel
 
     - name: Publish package
-      uses: pypa/gh-action-pypi-publish@master
+      uses: pypa/gh-action-pypi-publish@ec4db0b4ddc65acdf4bff5fa45ac92d78b56bdf0
```

```suggestion
      uses: pypa/gh-action-pypi-publish@ec4db0b4ddc65acdf4bff5fa45ac92d78b56bdf0 # v1.90
```
A pattern we've used elsewhere is to add the version tag as a comment. Dependabot is clever enough to understand this and keeps the comment up-to-date when it bumps the version.